docs/prometheus-integration.md

```diff
@@ -154,6 +154,10 @@ by flowPollInterval, a configuration parameter for the Agent.
 - **antrea_agent_denied_connection_count:** Number of denied connections
 detected by Flow Exporter deny connections tracking. This metric gets updated
 when a flow is rejected/dropped by network policy.
+- **antrea_agent_flow_exporter_to_flow_aggregator_count:** Number of Flow
+Exporters connected to Flow Aggregator. This metric gets updated when a
+connection is established or removed between Flow Exporter and Flow
+Aggregator.
```

Comment:
Is the expectation for this metric to be equal to the number of agents/nodes in the cluster? If so, it is better to add in the description.

Reply:
Actually the metric value will be either 0 or 1 from each antrea agent as @antoninbas pointed out. The sum of the metrics is expected to be number of agents. In that case are we supposed to go with current one or count the reconnections instead? 
![Screen Shot 2021-08-31 at 12 10 45 PM](https://user-images.githubusercontent.com/36459546/131561952-23578612-26a1-4864-b571-bf983bdb784d.png)

Reply:
Yup, that is correct; we are looking at per agent metrics. You could either do reconnections or maintain Prometheus metrics at the Flow Aggregator to track the connections to the exporter. 
Do you see any differences in above approaches?

Reply:
I think the number of connected agents is helpful, but for a given agent, it would also be helpful (e.g. to troubleshoot issues) to know the number of reconnections